lib/features/manage_account/presentation/always_read_receipt/always_read_receipt_view.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:tmail_ui_user/features/manage_account/presentation/menu/settings_utils.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+
+class AlwaysReadReceiptView extends GetWidget<AlwaysReadReceiptController> with AppLoaderMixin {
+  const AlwaysReadReceiptView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: SettingsUtils.getBackgroundColor(context, controller.responsiveUtils),
+      body: Container(
+        color: SettingsUtils.getContentBackgroundColor(context, controller.responsiveUtils),
+        decoration: SettingsUtils.getBoxDecorationForContent(context, controller.responsiveUtils),
+        margin: SettingsUtils.getMarginViewForSettingDetails(context, controller.responsiveUtils),
+        padding: SettingsUtils.getPaddingAlwaysReadReceiptSetting(context, controller.responsiveUtils),
+        child: Column(
+          children: [
+            Expanded(
+              child: ListView(
+                physics: const ClampingScrollPhysics(),
+                children: [
+                  Text(
+                    AppLocalizations.of(context).emailReadReceipts,
+                    style: const TextStyle(
+                      fontSize: 17,
+                      height: 24 / 17,
+                      fontWeight: FontWeight.w500,
+                      color: Colors.black)),
+                  const SizedBox(height: 8),
+                  Text(
+                    AppLocalizations.of(context).emailReadReceiptsSettingExplanation,
+                    style: const TextStyle(
+                      fontSize: 16,
+                      height: 20 / 16,
+                      color: AppColor.colorTextSettingDescriptions)),
+                  const SizedBox(height: 24),
+                  Row(
+                    children: [
+                      InkWell(
+                        onTap: controller.toggleAlwaysReadReceipt,
+                        child: Padding(
+                          padding: const EdgeInsets.all(4),
+                          child: Obx(() {
+                            return SvgPicture.asset(
+                              controller.alwaysReadReceipt.value
+                                ? controller.imagePaths.icSwitchOn
+                                : controller.imagePaths.icSwitchOff,
+                              fit: BoxFit.fill,
+                              width: 32,
+                              height: 20);
+                          }),
+                        ),
+                      ),
+                      const SizedBox(width: 12),
+                      Expanded(
+                        child: Text(
+                          AppLocalizations.of(context).emailReadReceiptsToggleDescription,
+                          style: const TextStyle(
+                            fontSize: 16,
+                            height: 20 / 16,
```

Comment:
why is that so `20 / 16`?

Reply:
Line height of the text here is 20. In Flutter, a text can be set line height to `x`, when it has font size `y`, with `height: x / y`. [See the docs](https://api.flutter.dev/flutter/painting/TextStyle/height.html)